src/test/ui/return/return-impl-maybe-sized.rs

```diff
@@ -0,0 +1,3 @@
+fn test_fn() -> impl ?Sized {}
```

Comment:
Could you add a comment like the below so that we can see why the test exists?

```suggestion
// Regression test for #97226

fn test_fn() -> impl ?Sized {}
```

Reply:
sorry, I just noticed this, already added